### PR TITLE
wasmedge: add version 0.13.5, add package_type

### PR DESCRIPTION
**Pull request by toge**
Specify library name and version:  **wasmedge/***

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/wasmedge/all/conanfile.py [hunk 1]
```diff
@@ -16,14 +16,15 @@ class WasmedgeConan(ConanFile):
...
     @property
     def _compiler_alias(self):
         return {
             "Visual Studio": "Visual Studio",
             "msvc": "Visual Studio",
-        }.get(str(self.settings.compiler), "gcc")
+        }.get(str(self.info.settings.compiler), "gcc")
 
     def configure(self):
         del self.settings.compiler.libcxx
```
Comment (RubenRBS) [inline recipes/wasmedge/all/conanfile.py:30]: msvc won't like this and will fail for windows

Comment (toge) [reply recipes/wasmedge/all/conanfile.py]: @RubenRBS 
Thanks a lot.
I fixed this problem.

**Comment by toge** [comment]
@RubenRBS 
Please review it again?
I think there are not any problems to merge it.

**Review by uilianries — APPROVED** [review]
LGTM

**Review by jwillikers — APPROVED** [review]

**Pull request merged**